web/yo/app/scripts/services/pubUtils.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+/**
+ * @ngdoc factory
+ * @name oncokbApp.pubUtils
+ * @author Jing Su
+ * @description
+ * This factory is a cache service used for storing validated PMIDs.
+ */
+angular.module('oncokbApp')
+    .factory('pubUtils', function() {
+        var validatedPubs = {};
+
+        function clear() {
+            validatedPubs = {};
+        }
+        function get(pubs) {
```

Comment:
pubs should just be a string array.

Reply:
I need to store the whole pub structure for not validated PMIDs later.